src/yuzu/main.cpp

```diff
@@ -1341,6 +1374,71 @@ void GMainWindow::OnGameListNavigateToGamedbEntry(u64 program_id,
     QDesktopServices::openUrl(QUrl(QStringLiteral("https://yuzu-emu.org/game/") + directory));
 }
 
+void GMainWindow::OnClearImportedSysdata() {
+    Core::System& system{Core::System::GetInstance()};
+
+    const auto path = FileUtil::GetUserPath(FileUtil::UserPath::SysDataDir) + "imported" DIR_SEP;
```

Comment:
```suggestion
    const auto path = FileUtil::GetUserPath(FileUtil::UserPath::SysDataDir) + "imported/";
```

Qt will automatically handle any necessary path conversions